Broken build due to reformatting

#109 (and others) seem to break the build.
During reformatting a lot of ```this.``` statements have been removed, leaving ambiguous symbols behind:
```error CS0136: A local variable named 'exp' cannot be declared in this scope because it would give a different meaning to 'exp', which is already used in a 'parent or current' scope to denote something else```
I'm compiling with VS2013.

--- a/Microsoft.Research/AbstractInterpretation/Abstract Domains/Array/ArraySegmentation.cs
+++ b/Microsoft.Research/AbstractInterpretation/Abstract Domains/Array/ArraySegmentation.cs
@@ -1761,11 +1761,11 @@ namespace Microsoft.Research.AbstractDomains
             }
 
             Contract.Assert(elements.Count >= 1);
-            Contract.Assert(limits.Count == elements.Count + 1);
+            Contract.Assert(this.limits.Count == elements.Count + 1);
 
             // Fast, syntactic search for the lower bound
             var found = false;
-            for (var i = 0; i < limits.Count - 1; i++)
+            for (var i = 0; i < this.limits.Count - 1; i++)
             {
                 if (limits[i].Contains(lowExp))
                 {
@@ -1779,7 +1779,7 @@ namespace Microsoft.Research.AbstractDomains
                 }
             }
 
-            Contract.Assert(limits.Count == elements.Count + 1);
+            Contract.Assert(this.limits.Count == elements.Count + 1);
 
             // Semantic search
             if (!found)

--- a/Microsoft.Research/CodeAnalysis/Expression.cs
+++ b/Microsoft.Research/CodeAnalysis/Expression.cs
@@ -4237,7 +4237,7 @@ namespace Microsoft.Research.CodeAnalysis
 
                     UnaryOperator op;
                     ExternalExpression arg;
-                    if (decoder.IsUnaryOperator(exp, out op, out arg))
+                    if (decoder.IsUnaryOperator(this.exp, out op, out arg))
                     {
                         var exp = BoxedExpression.For(arg, decoder);
 

